### DISCUSSION TITLE
can't download oracle drivers initializing new connection

### DISCUSSION THREAD

**Discussion by AVS-2307** [dbeaver/dbeaver#20437] [Q&A] [answered]
Hello all!

I've met with such kind of problem.
I tried to make new connection with Oracle DB and when I checked driver properties
![image](https://github.com/dbeaver/dbeaver/assets/103745537/ac3f7561-9ef1-43d5-8024-581d88ff1703)
I got the following error:
![image](https://github.com/dbeaver/dbeaver/assets/103745537/f85f45ad-92ea-44b4-a342-6da0df5a4306)
I marked and unmarked 'use windows trust store'
![image](https://github.com/dbeaver/dbeaver/assets/103745537/f6feacf7-48fe-4c38-af4f-07f379d2cb6e)
but it didn't help me.

I installed OracleInstantClinet and In driver manager I added a link to it.
![image](https://github.com/dbeaver/dbeaver/assets/103745537/fbf6f80c-cf8f-4b0c-8607-8e941d490f9c)

But all this actions didn't give any result, Oracle driver still unavalable.

Please, suggest me, what else should I do to get the connection?
Thank you!

**Comment by E1izabeth** [comment, marked as answer]
Did you restart DBeaver when you set/unset Use Windows trust store setting?

If you downloaded drivers on your own and added directory in the Driver -> Libraries, then you need to Remove blue items from that list.

**Reply by AVS-2307** [reply]
Yes, I made restart.
And the problem was in removing previous library links)
thank you so much!)

**Comment by AVS-2307** [comment]
If you downloaded drivers on your own and added directory in the Driver -> Libraries, then you need to Remove blue items from that list.
